### ISSUE TITLE
Infinite Scroll demo some views are hidden in ViewPort

### ISSUE THREAD

**Issue by msounthar** [runspired/smoke-and-mirrors#5]
Hi
I was just checking the infinite scroll demo here,
http://runspired.github.io/smoke-and-mirrors/#/infinite-scroll

I noticed some views are still being hidden when scrolled randomly up and down, even when they are in the viewport. The top view div is changing to visible as expected but its the inner view which is still being hidden.

It would be nice if you can look into this, as I am looking this as an alternate to ember-cloaking for my app.

Thanks.

**Comment by runspired** [comment]
This is just a factor of not having pushed a new demo, has been fixed for ages :D 

The blank views were the result of an attempt to cache the html output of views and re-hook them in later.  Unfortunately it turned out that could work in Ember 1.10 but not 1.11.  I pushed that work in to a feature branch and will revisit it if some of the fastboot work enables that approach to work again.

Pushing a new demo now.
